charts/aws-vpc-cni/values.yaml

```diff
@@ -84,6 +84,7 @@ env:
   ENABLE_IPv4: "true"
   ENABLE_IPv6: "false"
   VPC_CNI_VERSION: "v1.16.2"
+  ENABLE_ADV_ENI_CFG: "true"
```

Comment:
I think we need a name other than `ENABLE_ADV_ENI_CFG` here. `ENIConfig` is too closely related to Custom Networking, so this could get confusing.

Since at the end of the day we are expanding the pool of subnets we choose from, I think `ENABLE_ADV_SUBNET_SEL` is the best option. The pitch to customers is that we will automatically start using subnets they tag, so this seems the clearest variable to denote that.

Reply:
Agree, why do we call it *_ENI_CFG? Since we aren't using eniconfig right? Let's discuss internally on the naming..